.github/workflows/shared-build-deploy-container.yml

```diff
@@ -64,12 +54,12 @@ env:
   GITHUB_TOKEN: ${{ secrets.TRAKX_GITHUB_TOKEN || secrets.GITHUB_TOKEN }}
 
 jobs:
-  prepare:
+  prepare-and-validate:
     name: "Prepare the environment to run the jobs"
     runs-on: ubuntu-latest
```

Comment:
instead of ubuntu-latest we should use one of trakx runners
the reason is if github runners become unavailable, which happened before, all our pipelines will be blocked

Reply:
But in this step we don't know which runner to use. Do we have a generic one for Trakx?

Reply:
Created task TKX1091 to deal with this later.